include/TMS/TMS.hpp

```diff
@@ -30,18 +47,38 @@ class TMS {
 
 private:
     /**
-     * This sample data will be exposed over CAN through the object
-     * dictionary. The address of the variable will be included in the
-     * object dictionary and can be updated via SDO via a CANopen client.
-     * This device will then broadcast the value via a triggered PDO.
+     * Array to store the thermistor objects
      */
-    uint8_t sampleData;
+    DEV::Thermistor thermistors[NUM_THERMISTORS];
+
+    /**
+     * Array to store the thermistor values
+     */
+    uint32_t thermTemps[NUM_THERMISTORS];
+
+    /**
+     * Hardcoded conversion function from voltage to temperature in Celsius
+     * Only works for values up to 73.783 deg C
+     *
+     * @param adcInput Raw voltage in ADC units out of a maximum of 4095
+     * @return Temperature of the thermistor in Celsius
+     */
+    static uint32_t conversion(uint32_t adcInput) {
+        auto adc64 = (uint64_t) adcInput;
+        auto outTemp = (uint32_t) ((adc64 * adc64 * adc64) / 1000000000 * 9604
```

Comment:
I'd add a quick comment stating where the conversion function comes from so that someone in the future knows where to go if this needs to be updated.